temporalcli/commands.workflow_view_test.go

```diff
@@ -107,6 +108,53 @@ func (s *SharedServerSuite) TestWorkflow_Describe_Completed() {
 	s.Equal(map[string]any{"foo": "bar"}, jsonOut["result"])
 }
 
+func (s *SharedServerSuite) TestWorkflow_Describe_NotDecodable() {
+	s.Worker().OnDevWorkflow(func(ctx workflow.Context, input any) (any, error) {
+		return temporalcli.RawValue{
+			Payload: &common.Payload{
+				Metadata: map[string][]byte{"encoding": []byte("some-encoding")},
+				Data:     []byte("some-data"),
+			},
+		}, nil
+	})
+	// Start the workflow and wait until it has at least reached activity failure
+	run, err := s.Client.ExecuteWorkflow(
+		s.Context,
+		client.StartWorkflowOptions{TaskQueue: s.Worker().Options.TaskQueue},
+		DevWorkflow,
+		nil, // input is irrelevant
+	)
+	s.NoError(err)
+	s.NoError(run.Get(s.Context, nil))
+
+	// Text
+	res := s.Execute(
+		"workflow", "describe",
+		"--address", s.Address(),
+		"-w", run.GetID(),
+	)
+	s.NoError(res.Err)
+	out := res.Stdout.String()
+	s.ContainsOnSameLine(out, "Status", "COMPLETED")
+	s.ContainsOnSameLine(out, "ResultEncoding", "some-encoding")
+
+	// TODO: The below fails seemingly due to a marshalling bug in api-go. The conversion of the
+	//   returned raw payload in to JSON bytes is invalid json, and breaks the printing on this
```

Comment:
I'm going to take a look at fixing this in api-go

Reply:
NM, apparently this is already fixed, probably by https://github.com/temporalio/api-go/pull/159

Reply:
We just need to update versions when possible